src/Core/src/Core/ICanReturnVisualTreeElement.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Maui
+{
+	interface ICanReturnVisualTreeElement
```

Comment:
At a later point we can look at making this public for 3rd party folks but for now we'll keep it internal. 

Most folks I've chatted with start from our `MauiView` so usually that will work well enough